geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/RangeHistogram.scala

```diff
@@ -154,15 +154,16 @@ class RangeHistogram[T](val attribute: Int, initialBins: Int, initialEndpoints:
 object RangeHistogram {
 
   def buffer[T](value: T): (T, T) = {
```

Comment:
I'll add some scala docs - it's meant to take a single value and create a range from it.
When we build up stats, we take the current bounds to create the range histogram. But, if for a given attribute all the features have the same value, we wouldn't have a valid range to create a histogram with (because lower bound has to be less than the upper bound). So we take the single values and 'buffer' it when creating the histogram.
The histogram will expand as needed, so the initial bounds aren't too important. But the logic gets a lot simpler if you know that the lower bound will always be less than the upper bound.